openmetadata-spec/src/main/resources/json/schema/entity/data/dashboardDataModel.json

```diff
@@ -20,7 +20,8 @@
         "LookMlView",
         "LookMlExplore",
         "PowerBIDataModel",
-        "QlikSenseDataModel"
+        "QlikSenseDataModel",
+        "QlikCloudDataModel"
```

Comment:
would it make sense to have a `QlikDataModel` used for both qliksense and qlikcloud? Not sure how different the actual data is

Reply:
that would require a migration, but should be OK cc @ulixius9